frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts

```diff
@@ -139,7 +139,7 @@ describe('Bitbucket Service', () => {
 
   it('should detect Dockerfile', () => {
     const gitSource: GitSource = {
-      url: 'https://bitbucket.org/jerolimov/nodeinfo',
+      url: 'https://bitbucket.org/christoph-jerolimov/nodeinfo',
```

Comment:
Same here getting 404 for this. But old one is working fine https://bitbucket.org/jerolimov/nodeinfo

Reply:
Reverted this change